graphql-language/src/ast.rs

```diff
@@ -283,16 +282,11 @@ pub enum Type<'a> {
 }
 
 /// NamedType : Name
-pub struct NamedType<'a> {
```

Comment:
At least use the newtype pattern for this. In the reference implementation `NamedType` is a [node](https://github.com/graphql/graphql-js/blob/master/src/language/ast.js#L47) and has its own [type](https://github.com/graphql/graphql-js/blob/master/src/language/ast.js#L243-L247) (i.e. not a type alias). Let's stick to the reference implementation on this one.